packages/ember-htmlbars/lib/helper.js

```diff
@@ -41,6 +41,37 @@ import Object from 'ember-runtime/system/object';
 
   Additionally, class helpers can call `recompute` to force a new computation.
 
+  If the output of your helper is only dependent on the current input, then you
+  can use the `Helper.helper` function.
```

Comment:
@BryanCrotaz Perhaps this is where we need the crosslink that @mixonic mentioned

> See [Ember.Helper.helper](/api/classes/Ember.Helper.html#method_helper)